network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/formassembly_body.html

```diff
@@ -308,7 +310,7 @@
                 </div>
             {% endif %}
             <div class="oneField field-container-D  labelsAbove  " id="tfa_492-D" role="group" aria-labelledby="tfa_492-L" data-tfa-labelledby="-L tfa_492-L" aria-required="true">
-                <label id="tfa_492-L" class="label preField reqMark" aria-label="Privacy Agreement   required">Privacy Agreement</label><br><div class="inputWrapper"><span id="tfa_492" class="choices vertical required" aria-required="true"><span class="oneChoice"><input type="checkbox" value="tfa_493" class="calc-DefaultName" id="tfa_493" name="tfa_493" aria-labelledby="tfa_493-L" data-tfa-labelledby="tfa_492-L tfa_493-L"><label class="label postField" id="tfa_493-L" for="tfa_493"><span class="input-checkbox-faux"></span>{% blocktrans with link="https://www.mozilla.org/privacy/websites/" %}I’m okay with Mozilla handling my info as explained in this <a href="{{ link }}" target="_blank">Privacy Notice</a>.{% endblocktrans %}</label></span></span></div>
+                <label id="tfa_492-L" class="label preField" aria-label="Privacy Agreement  required">Privacy Agreement</label><br><div class="inputWrapper"><span id="tfa_492" class="choices vertical required" aria-required="true"><span class="oneChoice"><input type="checkbox" value="tfa_493" class="calc-DefaultName" id="tfa_493" name="tfa_493" aria-labelledby="tfa_493-L" data-tfa-labelledby="tfa_492-L tfa_493-L"><label class="label reqMark postField" id="tfa_493-L" for="tfa_493"><span class="input-checkbox-faux"></span>{% blocktrans with link="https://www.mozilla.org/privacy/websites/" %}I’m okay with Mozilla handling my info as explained in this <a href="{{ link }}" target="_blank">Privacy Notice</a>.{% endblocktrans %}</label></span></span></div>
```

Comment:
Moving the class `reqMark` to this element allows the asterisk to render to the right of the label.

Reply:
Could you add a new note (probably as Step 6) at the top of the file so we remember to make this change in future updates? Thanks!

Reply:
Hi Mavis! I have added a new comment [here](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/7038fa4851cfb5b4b1beb852d80cfc0983d013c6/network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/formassembly_body.html#L27-L29).